src/software/jetson_nano/display/screens/wheels_screen.py

```diff
@@ -0,0 +1,210 @@
+from screen import Screen
+
+FRONT_LEFT = 0
+FRONT_RIGHT = 1
+BACK_LEFT = 2
+BACK_RIGHT = 3
+
+PADDING = 6
+BASE_Y = 20
+
+"""
+This screen is used to edit wheel speed settings
+"""
+
+
+class WheelsScreen(Screen):
+    def __init__(self, lcd_display, redis_dict, status_codes):
```

Comment:
can we have some comments on this function?

Reply:
done